scripts/west_commands/runners/bossac.py

```diff
@@ -4,9 +4,11 @@
 
 '''bossac-specific runner (flash only) for Atmel SAM microcontrollers.'''
 
+import subprocess
```

Comment:
Please don't use subprocess directly; there are ZephyrBinaryRunner wrapper methods which should be used instead. You can use `self.check_output` instead of `subprocess.check_output`, for example.

This is important because the wrapper methods log the subprocess invocations, which is important  to make `west -v flash` usable for debugging when things go wrong with the underlying tool.

Reply:
Hi Marti.  The code already uses the wrappers, but imports subprocess to catch the subprocess.CalledProcessError thrown by running `bossac --help`

Please resolve if that's OK.

Reply:
Oh, I'm sorry for missing that. Thanks for explaining.